Makefile

```diff
@@ -112,78 +111,83 @@ delete-integration-test-dev-cluster: build ## Delete the test cluster for use wh
 
 ##@ Code Generation
 
-.PHONY: generate-bindata-assets
-generate-bindata-assets: ## Generate bindata assets (node bootstrap config files & add-on manifests)
-	chmod g-w  ./pkg/nodebootstrap/assets/*
-	env GOBIN=$(GOBIN) go generate ./pkg/nodebootstrap ./pkg/addons/default
+AWS_SDK_MOCKS := $(wildcard pkg/eks/mocks/*API.go)
+
+DEEP_COPY_HELPER := pkg/apis/eksctl.io/v1alpha5/zz_generated.deepcopy.go
+GENERATED_FILES := pkg/addons/default/assets.go \
+pkg/nodebootstrap/assets.go \
+$(DEEP_COPY_HELPER) \
+pkg/ami/static_resolver_ami.go \
+site/content/usage/20-schema.md \
+$(AWS_SDK_MOCKS)
+
+.PHONY: regenerate-sources
+# TODO: generate-ami is broken (see https://github.com/weaveworks/eksctl/issues/949 ), include it when fixed
```

Comment:
It take forever as it goes into each region and I don't think it need optimising, it should just ran as a periodic job.

Reply:
alright